### PR TITLE
docs: add docs for greenfield peer for BSC network

### PR DESCRIPTION
**Pull request by alexgao001**
add docs for greenfield peer for BSC network

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -219,6 +219,7 @@ nav:
               - Overview: ./bnb-greenfield/for-developers/data-archive/data-archive-layer.md
               - Blob Hub: ./bnb-greenfield/for-developers/data-archive/blob-hub.md
               - Block Archiver: ./bnb-greenfield/for-developers/data-archive/block-archiver.md
+              - Greenfield Peer: ./bnb-greenfield/for-developers/data-archive/greenfield-peer.md
```
Comment (unclezoro) [inline mkdocs.yml:222]: Greenfield peer or BSC peer?

File: docs/bnb-smart-chain/developers/node_operators/full_node.md [hunk 2]
```diff
@@ -139,6 +139,11 @@ The **full** sync mode can also be used to do initial sync, which will execute a
 
 If the flag **--syncmode** is not provided, the default sync mode will depend on the state of the data folder. It will be **snap** mode if you sync from genesis or **full** mode if you start from a snapshot.
 
+## Full Sync with Greenfield Peer (Optional)
+
+The Greenfield Peer is a data seed provided by Greenfield. You can config your BSC node to connect to the Greenfield Peer by adjusting the settings in your configuration file.
```
Comment (unclezoro) [inline docs/bnb-smart-chain/developers/node_operators/full_node.md:144]: can you hightlight that in what case people can configure greenfield peer, what is the usage?

File: docs/bnb-greenfield/for-developers/network-endpoint/endpoints.md [hunk 3]
```diff
@@ -91,3 +91,7 @@ order: 3
   - `https://opbnb-testnet.nodereal.io/v1/e9a36765eb8a40b9bd12e680a1fd2bc5`
 
   For more details, you can refer to [here](network-info.md).
+
+### Block Archiver
+- **BSC:**
```
Comment (unclezoro) [inline docs/bnb-greenfield/for-developers/network-endpoint/endpoints.md:96]: can you change the font in the `gnfd-peer.png`, we would like a more official font.

File: docs/bnb-greenfield/for-developers/network-endpoint/endpoints.md [hunk 4]
```diff
@@ -91,3 +91,7 @@ order: 3
...
+- `https://gnfd-bsc-archiver-testnet.bnbchain.org`
```
Comment (unclezoro) [inline docs/bnb-greenfield/for-developers/network-endpoint/endpoints.md:97]: can you also explain what the data is behind this endpoint?

File: docs/bnb-smart-chain/developers/node_operators/full_node.md (hunk 2 above)
Comment (alexgao001) [reply docs/bnb-smart-chain/developers/node_operators/full_node.md:144]: added

File: docs/bnb-greenfield/for-developers/network-endpoint/endpoints.md (hunk 4 above)
Comment (alexgao001) [reply docs/bnb-greenfield/for-developers/network-endpoint/endpoints.md:97]: added

**Review by unclezoro — APPROVED** [review]

**Pull request merged**